packages/design-system/src/types/Autocomplete/Autocomplete.d.ts

```diff
@@ -53,6 +53,10 @@ export interface AutocompleteProps extends Omit<DownshiftProps<any>, PropsNotPas
    * attribute on a label and the id of an input.
    */
   id?: string;
+  /**
+   * Customize the default status messages announced to screenreader users via aria-live when autocomplete results are populated. [Read more on downshift docs.](https://github.com/paypal/downshift#geta11ystatusmessage)
+  */
+  getA11yStatusMessage?: (...args: any[]) => any;
```

Comment:
Can we use downshift props here?
 i.e. `DownshiftProps<any>["onInputValueChange"];`
 
 The same can be done for other props like `itemToString`,

Reply:
Changed the DownshiftProps typing for `DownshiftProps` `getA11yStatusMessage`  `itemToString`